datumaro/cli/commands/describe_downloads.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (C) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import argparse
+import contextlib
+import sys
+from typing import Dict, Type
+
+from datumaro.components.extractor_tfds import (
+    AVAILABLE_TFDS_DATASETS,
+    TFDS_EXTRACTOR_AVAILABLE,
+    TfdsDatasetRemoteMetadata,
+)
+from datumaro.util import dump_json
+
+from ..util import MultilineFormatter
+
+
+def build_parser(
+    parser_ctor: Type[argparse.ArgumentParser] = argparse.ArgumentParser,
+):
+    parser = parser_ctor(
+        help="Print information about downloadable datasets",
+        description="""
+        Reports information about datasets that can be downloaded with the
+        "datum download" command. The information is reported either as
+        human-readable text (the default) or as a JSON object.
+        """,
+        formatter_class=MultilineFormatter,
+    )
+
+    parser.add_argument(
+        "--report-format",
+        choices=("text", "json"),
+        default="text",
+        help="Format in which to report the information (default: text)",
+    )
+    parser.add_argument(
+        "--report-file", help="File to which to write the report (default: standard output)"
+    )
+    parser.set_defaults(command=describe_downloads_command)
+
+    return parser
+
+
+def get_sensitive_args():
+    return {
+        describe_downloads_command: [],
+    }
+
+
```

Comment:
When I tried the command, I've got this output:
```
$ datum describe-downloads
2022-03-24 14:07:11,291 INFO: Load pre-computed DatasetInfo (eg: splits, num examples,...) from GCS: cifar10/3.0.2
2022-03-24 14:07:13,424 INFO: Load dataset info from /tmp/tmpce3r5izotfds
2022-03-24 14:07:13,426 INFO: Field info.citation from disk and from code do not match. Keeping the one from code.
2022-03-24 14:07:13,426 INFO: Field info.splits from disk and from code do not match. Keeping the one from code.
2022-03-24 14:07:13,426 INFO: Field info.supervised_keys from disk and from code do not match. Keeping the one from code.
2022-03-24 14:07:13,426 INFO: Field info.module_name from disk and from code do not match. Keeping the one from code.
...
```

I think, it should be managed by `--loglevel`, and also it should be classified as debug output.

Reply:
> I think, it should be managed by --loglevel

It _is_ managed by `--loglevel`.

> and also it should be classified as debug output.

It comes from TFDS. I can't affect the log levels it uses.